src/overlays/actors/ovl_En_Anubice/z_en_anubice.c

```diff
@@ -292,52 +299,52 @@ void EnAnubice_SetupDie(EnAnubice* this, GlobalContext* globalCtx) {
     this->animLastFrame = lastFrame;
     Animation_Change(&this->skelAnime, &gAnubiceFallDownAnim, 1.0f, 0.0f, lastFrame, ANIMMODE_ONCE, -20.0f);
 
-    this->unk_256 = false;
-    this->unk_258 = 0;
+    this->isFallingOver = false;
+    this->fallTargetPitch = 0;
     this->deathTimer = 20;
     this->actor.velocity.x = this->actor.velocity.z = 0.0f;
     this->actor.gravity = -1.0f;
 
     if (BgCheck_SphVsFirstPoly(&globalCtx->colCtx, &this->fireballPos, 70.0f)) {
-        this->unk_256 = true;
-        this->unk_258 = this->actor.shape.rot.x - 0x7F00;
+        this->isFallingOver = true;
+        this->fallTargetPitch = this->actor.shape.rot.x - 0x7F00;
     }
 
     this->actionFunc = EnAnubice_Die;
 }
 
 void EnAnubice_Die(EnAnubice* this, GlobalContext* globalCtx) {
     f32 curFrame;
-    f32 phi_f2;
-    Vec3f sp4C = { 0.0f, 0.0f, 0.0f };
+    f32 rotX;
+    Vec3f fireEffectInitialPos = { 0.0f, 0.0f, 0.0f };
     Vec3f fireEffectPos = { 0.0f, 0.0f, 0.0f };
     s32 pad;
 
     SkelAnime_Update(&this->skelAnime);
     Math_ApproachZeroF(&this->actor.shape.shadowScale, 0.4f, 0.25f);
 
-    if (this->unk_256) {
-        Math_SmoothStepToS(&this->actor.shape.rot.y, this->unk_258, 1, 10000, 0);
-        if (fabsf(this->actor.shape.rot.y - this->unk_258) < 100.0f) {
-            this->unk_256 = false;
+    if (this->isFallingOver) {
+        Math_SmoothStepToS(&this->actor.shape.rot.y, this->fallTargetPitch, 1, 10000, 0);
```

Comment:
So `fallTargetPitch` is initialized using rot.x but is used to change rot.y :thinking:

Are you sure about this falling over thing?

Reply:
Hmm, I'm probably mixing up my x and y rotation. Either way it's a bit odd...